types/graphql-list-fields/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for graphql-list-fields 2.0
+// Project: https://github.com/jakepusateri/graphql-list-fields#readme
+// Definitions by: Chris Filipowski <https://github.com/filipows>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { GraphQLResolveInfo } from "graphql";
+
+declare namespace getFieldList {}
```

Comment:
It seems the empty namespace can be removed.

Reply:
I added this line in order to allow importing module with `import * as syntax`. For ex:
`import * as getFieldNames from 'graphql-list-fields';`

Reply:
The empty namespace is a hack, and discouraged: https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
Cheers Mate! You're right. I knew it's a hack, but I found 100 packages in DefinitelyTyped repo, that use empty namespaces, that's why I thought that maybe it's accepted. Apparently, as I see now, I was wrong and it actually shouldn't work as it's against the es6 standard. I updated the PR.